src/pybind/ceph_argparse.py

```diff
@@ -1024,6 +1085,7 @@ def validate(args, signature, flags=0, partial=False):
                     validate_one(kwarg_v, kwarg_desc)
                     matchcnt += 1
                     store_arg(kwarg_desc, d)
+                    matchcnt += 1
```

Comment:
Incrementing matchcnt is now redundant, back when the patch was first created it was required. But since I've pushed a PR that has since merged that did just this to fix  https://github.com/ceph/ceph/pull/33004 (now just needs to be back ported)